README.md

```diff
@@ -1,69 +1,118 @@
 # CsvToAsset
-> Migrate legacy PAM assets to new Asset Manager assets (_available since 3.2_), using CSV file import.
 
-# Installation
-First, clone this repository, then:
+This tool migrates legacy PAM assets (_before 4.0_) to new Asset Manager assets (_available since 3.2_).
```

Comment:
I don't think we should say it's available since 3.2, It was not activated by default so I'm a bit "partagé" on stating this here.